test/QuadForm/Quad/Genus.jl

```diff
@@ -149,11 +149,20 @@
   OF = maximal_order(F)
   pl = real_places(F)
   p = prime_ideals_over(OF, 2)[1]
+  p3 = prime_ideals_over(OF, 3)[1]
+
+  # Test the computation of jordan decomposition
   G = Hecke.local_genera_quadratic(F, p, rank = 3, det_val = 2)
   for g in G
     g1 = genus(QuadLat, p, g.ranks, g.scales, g.weights, g.dets, g.normgens, g.witt)
     representative(g1) in G  # computes jordan decompositions
   end
+  G = Hecke.local_genera_quadratic(F, p3, rank = 3, det_val = 2)
+  for g in G
+    g1 = genus(QuadLat, p3, g.uniformizer, g.ranks, g.scales, g.detclasses)
+    representative(g1) in G  # computes jordan decompositions
```

Comment:
This won't run, there are typos in the name of the functions lines 1516 and 1522 in src/QuadForm/Quad/Genus.jl; it is written " _local_jordan_decomposi[[**s**]]tions_nondyadic! "

Reply:
Thanks